src/coresyftools/coresyftools/gpt_tool.py

```diff
@@ -55,22 +63,24 @@ def run(self, bindings):
         bindings = bindings.copy()
         operator = self.operation.get('operation')
         if 'graph' in self.operation and self.operation['graph']:
-            graph_file = os.path.join(self.context_directory, self.DEFAULT_GPT_GRAPH_FILE_NAME)
+            graph_file = os.path.join(self.context_directory,
+                                      self.DEFAULT_GPT_GRAPH_FILE_NAME)
```

Comment:
I remember in last sprint retrospective we discussed something about linting changes in separate commits :stuck_out_tongue: